complaint_search/views.py

```diff
@@ -76,13 +85,29 @@ def suggest(request):
     data = { k:v for k,v in request.query_params.iteritems() if k in QPARAMS_VARS }
     serializer = SuggestInputSerializer(data=data)
     if serializer.is_valid():
-        results = es_interface.suggest(**serializer.validated_data)
+        try:
+            results = es_interface.suggest(**serializer.validated_data)
+        except TransportError as e:
+            status_code = e.status_code if isinstance(e.status_code, int) \
+                else status.HTTP_400_BAD_REQUEST
+            res = {
+                "error": 'Elasticsearch error: ' + e.error
+            }
+            return Response(res, status=status_code)
         return Response(results)
     else:
         return Response(serializer.errors,
             status=status.HTTP_400_BAD_REQUEST)
 
 @api_view(['GET'])
 def document(request, id):
-    results = es_interface.document(id)
+    try:
+        results = es_interface.document(id)
+    except TransportError as e:
+        status_code = e.status_code if isinstance(e.status_code, int) \
+            else status.HTTP_400_BAD_REQUEST
+        res = {
+            "error": 'Elasticsearch error: ' + e.error
+        }
+        return Response(res, status=status_code)
```

Comment:
This block is essentially duplicated three times.  May want to rewrite as a wrapper function?  Or a decorated method?